src/main/scala/castalia/metrics/MetricsCollectorActor.scala

```diff
@@ -0,0 +1,32 @@
+package castalia.metrics
+
+import akka.actor.{Actor, ActorLogging, Props}
+import castalia.model.Messages.{EndpointCalled, EndpointMetricsInit, EndpointMetricsGet}
+import castalia.model.Model.EndpointMetrics
+
+object MetricsCollectorActor {
+
+  val metricNumberOfCalls = "calls"
+
+  def props: Props = Props(new MetricsCollectorActor)
+}
+
+/**
+  * Actor that aggregates various metrics specific to endpoints.
+  */
+class MetricsCollectorActor extends Actor with ActorLogging {
+
+  import MetricsCollectorActor._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/scala-academy/Castalia/file/1964138210/issues/source?bid=2998213&fileBranchId=3111257#l19)